apps/users/schemas.py

```diff
@@ -221,6 +221,19 @@ class CreateStudentOut(BaseOutSchema):
     student_id: int
 
 
+class CreateStudentsOut(BaseOutSchema):
+    student_id: int
+    first_name: str
+    last_name: str
+    middle_name: str = None
+    telephone_number: str
+    gender: str
+    course_id: int
+    speciality_id: int
+    user_id: None
```

Comment:
in swagger: "user_id": "Unknown Type: null", =)
u forgot to add type of column user_id

Reply:
ok

Reply:
fixed